make/config.mk

```diff
@@ -216,6 +216,7 @@ USE_CPP_PACKAGE = 0
 
 # Use int64_t type for tensor size.
 # This will cause performance degradation reported in issue #14496
+# Set to 1 for large tensor size greater than 2 billion
```

Comment:
not to be too picky, but what does this mean "tensor size greater than 2 billion"? Does this mean that if you have more than 2 billion elements in a single dimension you need this? or 2 billion elements in the whole tensor?

Reply:
updated